root/types/entity.js

```diff
@@ -41,7 +55,7 @@ declare type CollectableCoreEntityT =
   | WorkT;
 
 declare type NonUrlCoreEntityT =
-  | CollectableCoreEntityT
+  | CollectableEntityT
   | GenreT;
```

Comment:
I'm not sure I like defining this in terms of collectable entities to begin with, but it's unrelated to the PR.

Reply:
Completely agree on that.